doc/ssl-documentation.txt

```diff
@@ -0,0 +1,70 @@
+# Setting up an SSL hub
+
+## About certificates
+
+Before you can setup an SSL protected hub, you must create an SSL certificate for the hub.
+NOTE: uhub must be compiled with SSL support enabled in order for this to work (enabled by default, but not for Windows).
+
+## Configuring uhub
+
+If you have your certificates ready, just set these configuration values in uhub.conf file:
+
+  tls_private_key="/path/to/domainname.key"
```

Comment:
Thank you very much for this change. But the quotes should be removed from this line. The TLS keys are not read if there are quotes in this path.

Reply:
I use with quote and it works